superset/assets/javascripts/explorev2/components/controls/Filter.jsx

```diff
@@ -70,13 +82,13 @@ export default class Filter extends React.Component {
         this.fetchFilterValues(filter.col);
       }
     }
-    if (this.props.having) {
-      // druid having filter
+    if (this.props.having || strFilterOps.indexOf(filter.op) !== -1) {
```

Comment:
seems like we  may not need `this.props.having ||` anymore but I may be missing something